src/makeRobotAccountant.js

```diff
@@ -6,7 +6,25 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let callCounter = 0;
+
+  return function(a) {
+    callCounter++;
+
+    if (callCounter <= 3) {
+      return function(b) {
+        return a + b;
+      };
+    } else if (callCounter % 2 === 0) {
+      return function() {
+        return 'Bzzz... Error!';
+      };
+    } else {
+      return function(b) {
```

Comment:
`return function(b)`  
return this function once, without if else and perform operations in the middle
I mean,  
  return (a) => {
    return (b) => { 
    }
   }